terraform/modules/paas/main.tf

```diff
@@ -76,18 +76,24 @@ resource cloudfoundry_app worker_app {
 }
 
 resource cloudfoundry_route web_app_cloudapps_digital_route {
+  count = var.disable_routes == null ? 1 : 0
```

Comment:
Do we need to move the cloudapps route?

Reply:
If they want to test a review app via the find route, then I'd think so. Also seems to be a risk to leave a route that would remain pointing to find.

Reply:
As discussed, this is required for review apps